browser/src/ClinvarVariantsTrack/ClinvarAllVariantsPlot.tsx

```diff
@@ -216,15 +216,19 @@ const VariantLine = ({
     const frameshiftMaxPos = Math.max(endpoint1, endpoint2)
     const terminationPos =
       transcript && transcript.strand === '+' ? frameshiftMaxPos : frameshiftMinPos
-    // @ts-expect-error TS(2532) FIXME: Object is possibly 'undefined'.
-    const frameshiftExonRegions = transcript.exons
-      .sort((e1, e2) => e1.start - e2.start)
-      .filter((e) => e.start <= frameshiftMaxPos && e.stop >= frameshiftMinPos)
-      .map((e) => ({
-        start: Math.max(e.start, frameshiftMinPos),
-        stop: Math.min(e.stop, frameshiftMaxPos),
-        feature_type: e.feature_type,
-      }))
+
+    // if a frameshift variant-consequence pair from ClinVar exists for a transcript
```

Comment:
So this fix is fine far as it goes, but I'd suggest you also do two other things:

1. add a type annotation a couple lines above here reflecting that `transcript` might be undefined, i.e. change line 213 to something like: `const transcript: Transcript | undefined = ...`. That way we won't get bitten here by a similar case in the future.
2. Update `getGlobalFrameshiftCoordinates` also to reflect that `transcript` might be `undefined`, the logic might be fine as-is but a type annotation on there would be good in any case.

Reply:
Aha very good call, I didn't think to add a type annotation on the variable itself. 

I added a fixup commit to address point 1.

Point 2. is already addressed before this PR (getGlobalFrameshift... expects the possibility of `undefined` and handles it if so), so that's why theres not fixup commit to address that.